### PR TITLE
Blue block grid styles

### PR DESCRIPTION
**Pull request by kathleenAMA**
:art:

Adding in lots of coolio styles. 

 - Added in AMA's fancy new icon font!! Woo. 
 - Added in some more helper classes for thingsl ike buttons and bg colours
 - Added in styles for graph (currently only in automotive) using BEMish naming 
 - Removed and changed a few styles related to the new blue boxes that use the grid

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/webfonts/ama-icon-font.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+@font-face {
+  font-family: "ama-icon-font";
+  src:asset-url("ama_layout/webfonts/fonts/ama-icon-font.eot");
+  src:asset-url("ama_layout/webfonts/fonts/ama-icon-font.eot?#iefix") format("embedded-opentype"),
+    asset-url("ama_layout/webfonts/fonts/ama-icon-font.woff") format("woff"),
+    asset-url("ama_layout/webfonts/fonts/ama-icon-font.ttf") format("truetype"),
+    asset-url("ama_layout/webfonts/fonts/ama-icon-font.svg#ama-icon-font") format("svg");
+  font-weight: normal;
+  font-style: normal;
+
```
Comment (adamrmelnyk) [inline app/assets/stylesheets/ama_layout/webfonts/ama-icon-font.scss]: Do you need this extra space here?

**Comment by adamrmelnyk** [comment]
💩

**Comment by sineade** [comment]
:art: :art: :art:

**Comment by kathleenAMA** [comment]
:wave:

**Comment by sineade** [comment]
:art: :art: :art:

**Pull request merged**